src/modules/rlm_sql/drivers/rlm_sql_postgresql/rlm_sql_postgresql.c

```diff
@@ -74,6 +74,26 @@ static const CONF_PARSER driver_config[] = {
 	CONF_PARSER_TERMINATOR
 };
 
+#define USEC 1000000
+static void tv_sub(struct timeval *end, struct timeval *start,
```

Comment:
Hmm, this doesn't exist already?

Reply:
Given the care for the master patch, I'm going to assume it doesn't, and humbly request this gets added to the util library.  I would just double-check and grep for uses of struct timeval, because I really thought we had timeval arithmetic functions in v3.

Reply:
So we're never short of a function like this when we need one!...

Reply:
Wow, was that three or four duplications of the same function *sigh*